### PR TITLE
Replace single line java comments with multi-line style comments

### PR DESCRIPTION
**Pull request by SivaMaplelabs**

### CODE REVIEW

File: platform/src/main/java/org/hillview/dataset/api/IJson.java [hunk 1]
```diff
@@ -28,9 +28,10 @@
 import java.lang.reflect.Type;
 import java.time.Instant;
 
-// Unfortunately this module introduces many circular dependencies, because it has
-// to register various type adaptors.
-
+/**
```
Comment (mbudiu-vmw) [inline platform/src/main/java/org/hillview/dataset/api/IJson.java]: This is not a proper javadoc comment. a Javadoc comment should describe what the described object is doing, and not what the shortcomings are. This should stay as a regular comment.

File: platform/src/main/java/org/hillview/dataset/api/IMonoid.java [hunk 2]
```diff
@@ -32,8 +32,11 @@
 
     @Nullable
     default R reduce(List<R> data) {
-        // This implementation avoids allocating a zero
-        // if the list is non-empty.
+
+        /**
```
Comment (mbudiu-vmw) [inline platform/src/main/java/org/hillview/dataset/api/IMonoid.java]: Don't use JavaDoc comments for code, only for fields, classes, etc.

Comment (SivaMaplelabs) [reply platform/src/main/java/org/hillview/dataset/api/IMonoid.java]: Sure

**Comment by SivaMaplelabs** [comment]
@mbudiu-vmw Have addressed the comments.

**Pull request merged**